sdk/servicebus/azure-messaging-servicebus-stress/pom.xml

```diff
@@ -0,0 +1,97 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-messaging-servicebus-stress</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-servicebus-stress;current} -->
+  <packaging>jar</packaging>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+    <!-- Always target the minimum supported Java release in stress tests. -->
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+
+    <!-- Skip all linting in stress tests. -->
+    <codesnippet.skip>true</codesnippet.skip>
+    <checkstyle.skip>true</checkstyle.skip>
```

Comment:
Do we need to skip all these? I feel like we should check in "good" code regardless of whether it is shipped or not because people may try to understand our code later when they add more tests.

Reply:
Here I refer the perf test pom.xml. I can try to remove skip part to see if I can pass all checks.

Reply:
Removed skip.